### ISSUE TITLE
[RemoteRetro/production] SyntaxError: Unexpected end of JSON input

### ISSUE THREAD

**Issue by vanderhoop** [stride-nyc/remote_retro#439]
## Backtrace

[View full backtrace and more info at honeybadger.io](https://app.honeybadger.io/projects/54737/faults/37890960)

**Comment by mike-ap** [comment]
@vanderhoop , are these backtraces public?

**Comment by vanderhoop** [comment]
hey @mike-ap! sorry for the delayed response. can't expose the _full_ honeybadger page, but here's what we can expose:
```
category_column.jsx  38  55 parse(...)
webpack:///./web/static/js/components/category_column.jsx:38:55:in `parse'
36     event.preventDefault()
37     const { category, retroChannel } = this.props
38     const { id, body, assignee_id: assigneeId } = JSON.parse(event.dataTransfer.getData("idea"))
```

My guess is that an unexpected drag event (maybe of an image) triggered this, as that wouldn't have had an equivalent `setData` call. @ianmcnally you got any ideas?

**Comment by vanderhoop** [comment]
@mike-ap @ianmcnally looked into it, and was able to reproduce the error with a drag of an image into a category_column:

![triggering_json_parse_bug](https://user-images.githubusercontent.com/4386645/41949716-d52b8abe-797f-11e8-9690-e0134d1f3b60.gif)

We could disable drag events on the images via CSS, or ensure that the call to `event.dataTransfer.getData("idea")` returns a non-empty string, short-circuiting otherwise. I think the latter is the better long-term approach, as that prevents _all_ future events lacking `idea` data from triggering the error

**Comment by ianmcnally** [comment]
agreed with the long term approach. i thought a drop wasn't allowed on a non-draggable item.
